install_config/install/advanced_install.adoc

```diff
@@ -1181,6 +1181,139 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
 openshift_hosted_logging_storage_kind=dynamic
 ----
 
+[[enabling-service-catalog]]
+=== Enabling the Service Catalog
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+====
+
+Enabling the
+xref:../../architecture/service_catalog/index.adoc#architecture-additional-concepts-service-catalog[service catalog] allows service brokers to be registered with the catalog. The web
+console is also configured to enable an updated landing page for browsing the
+catalog.
+
+To enable the service catalog, add the following in your inventory file's
+`[OSEv3:vars]` section:
+
+----
+openshift_enable_service_catalog=true
+ifdef::openshift-enterprise[]
+openshift_service_catalog_image_prefix=openshift3/ose-
+endif::[]
+ifdef::openshift-origin[]
+openshift_service_catalog_image_prefix=openshift/origin-
+openshift_service_catalog_image_version=latest
+endif::[]
+----
+
+When the service catalog is enabled, the web console shows the updated landing
+page but still uses the normal image stream and template behavior. To deploy and
+register the provided service brokers, see
+xref:configuring-ansible-service-broker[Configuring the Ansible Service Broker]
+and xref:configuring-template-service-broker[Configuring the Template Service Broker].
+
+[[configuring-ansible-service-broker]]
+=== Configuring the Ansible Service Broker
+
+[NOTE]
+====
+The Ansible service broker is currently a Technology Preview feature.
+====
+
+If you have xref:enabling-service-catalog[enabled the service catalog], you can
+also enable the
+xref:../../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[Ansible service broker] (ASB).
+
+The ASB deploys its own etcd instance separate from the etcd used by the rest of
+the {product-title} cluster. The ASB's etcd instance requires separate storage
+using persistent volumes (PVs) to function, as well as for the Ansible playbook
+bundles (APBs) that are created using the ASB. If no PV is available, etcd will
+wait until the PV can be satisfied. The ASB application will enter a `CrashLoop`
+state until its etcd instance is available.
+
+[NOTE]
+====
+The following example shows usage of an NFS host to provide the required PVs,
+but
+xref:../../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[other persistent storage providers] can be used instead.
+====
+
+To enable the ASB and define its storage requirements:
+
+. Before running the installer, set up an NFS host.
+
+. Add the following in your inventory file's `[OSEv3:vars]` section:
++
+----
+ifdef::openshift-enterprise[]
+ansible_service_broker_image_prefix=openshift3/
```

Comment:
@fabianvf does this prefix need to be openshift3/ose-
or is that handled in a different spot?